incremental-font-transfer/src/uri_templates.rs

```diff
@@ -65,13 +66,58 @@ impl std::error::Error for UriTemplateError {}
 /// uri templates. Notably, only level one substitution expressions are supported and there are a fixed
 /// set of variables used in the expansion (id, id64, d1, d2, d3, and d4).
 ///
-/// All arguments are assumed to be utf8 encoded strings.
+/// template_string is assumed to be a utf8 encoded string.
```

Comment:
this isn't really an assumption, it's an invariant (there's no way to get a non-utf8 `&str` without unsafe)